Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM docker.io/library/rust:1.62.0@sha256:8220e4fbb22a07b78e6472cdf8f5fb8913a04974c26b130177b73a8a64334541 as builder
+FROM docker.io/rust:latest as builder
```

Comment:
Pinning the builder image to a specific @sha256:hash blocks using the docker container on MacOS Arm64 - by using rust:latest & debian:latest the container is able to be used on MacOS Arm64 - Docker selects the appropriate architecture.